llvm/include/llvm/TargetParser/X86TargetParser.def

```diff
@@ -257,6 +257,8 @@ X86_FEATURE_COMPAT(USERMSR,         "usermsr",                0)
 X86_FEATURE_COMPAT(AVX10_1,         "avx10.1-256",           36)
 X86_FEATURE_COMPAT(AVX10_1_512,     "avx10.1-512",           37)
 X86_FEATURE       (ZU,              "zu")
+X86_FEATURE_COMPAT(AVX10_2,         "avx10.2-256",            0)
```

Comment:
should be before "zu"

Reply:
and pls update clang/test/CodeGen/target-builtin-noerror.c

Reply:
Done.